### PR TITLE
Add padding to product logos

### PR DESCRIPTION
**Pull request by hasparus**
Need 2 pixels of space here, so it doesn't look like the logo is too large.

image

### CODE REVIEW

File: packages/components/src/products.tsx [hunk 1]
```diff
@@ -270,12 +270,14 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
         newWindow: true,
         title: (
           <div className="flex items-center gap-2">
-            <Logo
+            <div
               className={cn(
                 'size-4 translate-y-[0.25px]',
```
Comment (dimaMachina) [inline packages/components/src/products.tsx:275]:
```suggestion
                'translate-y-[0.25px]',
```

File: packages/components/src/products.tsx [hunk 2]
```diff
@@ -270,12 +270,14 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
...
-                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10',
+                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10 p-0.5',
```
Comment (dimaMachina) [inline packages/components/src/products.tsx:276]:
```suggestion
                i > 3 && 'rounded-sm bg-gray-500 text-white dark:bg-white/10 p-0.5',
```
not text inside

Comment (hasparus) [reply packages/components/src/products.tsx:276]: Oh yeah, this should be lower there on the logo, good catch.

Comment (hasparus) [reply packages/components/src/products.tsx:276]: We want `text-[8px]` to override the `text-sm`.

image

Comment (dimaMachina) [reply packages/components/src/products.tsx:276]: ah yeah I forgot that we have text (not SVG) logos

File: packages/components/src/products.tsx [hunk 3]
```diff
@@ -270,12 +270,14 @@ export const PRODUCTS_MENU_LIST: MenuItem['items'] = Object.fromEntries(
...
+            <div
               className={cn(
-                'size-4 translate-y-[0.25px]',
-                i > 3 && 'rounded-sm bg-gray-500 text-[8px] text-white dark:bg-white/10',
+                'flex size-4 translate-y-[0.25px] items-center justify-center',
+                i > 6 && 'rounded-sm bg-gray-500 text-white dark:bg-white/10',
               )}
-            />
+            >
+              <Logo className="size-4 text-[8px]" />
```
Comment (dimaMachina) [inline packages/components/src/products.tsx:279]: this piece of code is overcomplicated to me

the container defines `flex size-4 items-center justify-center` but children have `size-4` so technically you can remove everything from the container

and `translate-y-[0.25px]` do human eye see this difference of 1/4 pixel?

File: packages/components/src/components/tools-and-libraries-cards/index.tsx [hunk 4]
```diff
@@ -29,7 +34,7 @@ export function ToolsAndLibrariesCards({ className }: { className?: string }) {
       </Heading>
       <p className="text-green-800">Complete GraphQL Federation Stack</p>
       <ul className="-mx-12 -my-2 flex grid-cols-2 flex-row gap-[22px] overflow-auto px-12 py-2 lg:grid xl:grid-cols-4">
```
Comment (dimaMachina) [inline packages/components/src/components/tools-and-libraries-cards/index.tsx:36]:
```suggestion
      <ul className="-mx-12 -my-2 flex grid-cols-2 gap-[22px] overflow-auto px-12 py-2 lg:grid xl:grid-cols-4">
```

File: packages/components/src/products.tsx (hunk 3 above)
Comment (hasparus) [reply packages/components/src/products.tsx:279]: > and translate-y-[0.25px] do human eye see this difference of 1/4 pixel?

it was uneven to me without it. Take a look at the video.

https://github.com/user-attachments/assets/1e2cd05e-67bb-4314-89ae-e9c3182749cb

I have a 2021 MacBook with 14.2-inch (diagonal) Liquid Retina XDR display; 3024x1964 native resolution at 254 pixels per inch, but I'm running it at 1512x982 resolution, so at least every 0.5px should be visible, but empirically I feel it's 0.25px. 

You an go lower, but AFAIK browsers can do some rounding and the effects may differ across engines.

https://www.simonbattersby.com/blog/browsers-and-fractional-pixels/

---

> so technically you can remove everything from the container

Good catch, not everything though.
This is how it looks when you remove everything.

image
image

It's still fine if we remove everything but `flex`. We need it for the box not to grow to 24px height.

**Review by dimaMachina — APPROVED** [review]

**Pull request merged**